README.md

```diff
@@ -9,7 +9,153 @@
 
 `./gradlew run`
 
-## Ports
+## Queries
+Here are some of the important GraphQL queries:
+
+### 1. Get all traces in provided time range
+
+```graphql
+curl -s localhost:2020/graphql -H 'Content-Type: application/graphql' -d\
```

Comment:
FWIW, I think it's pretty uncommon to use curl to interface with graphql, although of course workable. I would suspect most readers would use a graphql client. That is to say, the curl part of the doc feels like just noise to me and I think we can just document the queries.

Reply:
Until we have a proper user capable api, I think this is better than nothing as currently people use the zipkin or jaeger apis to poke their systems

We will be using the trace ID and the trace in time range ones for testing, so these two are helpful for examples. It took a very long time to even figure these out. I actually was unable to find anyone who used GraphQL before.

Granted some of the ancillary, repetitive queries could easily be grouped up. Ex get entity name: example of api, endpoint, backend. Agree on that for sure.. pruning sounds fine!

Reply:
Definitely better than nothing! I guess it gets more to the purpose here - if trying to give a one liner sanity check for things like a health check, curl makes sense- everyone has it and can just copy paste. 

If trying to actually work with graphql, there are many interactive clients that are much easier to work with and make things discoverable, include the schema etc. So I guess the primitives I'd expect in the docs are the queries more than the curl commands. Anyway, a nit.

![image](https://user-images.githubusercontent.com/45047841/91240892-1e215400-e6f8-11ea-9f1e-2004bbaa96ee.png)